syntax = "proto3";

package bakery;

message OrderRequest {
  string order = 1;
}

message OrderResponse {
  string order_status = 1;
}

service Bakery {
  rpc GetOrder(OrderRequest) returns (OrderResponse) {}
}